tests/publiccloud/run_ltp.pm

```diff
@@ -79,18 +79,16 @@ sub run {
         $instance->run_ssh_command(cmd => 'sudo zypper -q in -y ltp',                              timeout => 600);
     }
 
-    assert_script_run('git clone -q --single-branch -b runltp_ng_openqa --depth 1 https://github.com/cfconrad/ltp.git');
-
+    assert_script_run('git clone -q --single-branch -b master --depth 1 https://github.com/metan-ucw/runltp-ng.git');
```

Comment:
I'd rather have a cloned repo of the upstream variant that we control, otherwise we are using foreign code that we don't control.

Why not making this a setting?
```suggestion
    my $ltp_repo = get_var("LTP_REPO", "https://github.com/INSERTHERETHEREPO");
    assert_script_run("git clone -q --single-branch -b master --depth 1 $ltp_repo");
```

Then before we update the repo, we can do a verification run by setting `LTP_REPO=https://github.com/metan-ucw/runltp-ng.git` and if it passes, we update the repo.

Reply:
i would rather keep it as it is and if we see problem we can adjust. i cant really see a strong reason for what you propose. the idea was to have the upstream code instead of the forked. After the changes in the `metan-ucw/runltp-ng.git` we should not see any difference.

Reply:
please check https://github.com/metan-ucw/runltp-ng.git commit history , it is not that project is in super active development . Also knowing main contributors I would say that chances that we get some commit which will brake functionality are really low, so I would say we still in the save side . But obviously no one can give 100% guaranty here :) . So I would follow your suggestion , but keep metan's repo as default. Also AFAIK `LTP_REPO` variable already have [different purpose](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/publiccloud/run_ltp.pm#L50) so we can not use it  : 
```suggestion
    my $ltp_repo = get_var("LTP_RUN_NG_REPO", "https://github.com/metan-ucw/runltp-ng.git");
    assert_script_run("git clone -q --single-branch -b master --depth 1 $ltp_repo");
```

WDYT @b10n1k  ?

Reply:
I would also like to have a LTP_RUN_NG_REPO variable.

Reply:
Ok, then it's good. If we could still have a customizable variable then it should be safe to do so!

Reply:
no objection

Reply:
changes on 871addf1d